Cabal/tests/PackageTests/Tests.hs

```diff
@@ -120,10 +120,6 @@ tests config = do
   -- with the same name and LATER version
   tc "BuildDeps/InternalLibrary3" $ internal_lib_test "internal"
 
-  -- Test that an explicit dependency constraint which doesn't
```

Comment:
And I guess the files should be deleted as well, together with the `extra-source-files` entry?